CIME/scripts/create_test.py

```diff
@@ -361,6 +365,15 @@ def parse_command_line(args, description):
         "\noption will allow existing baseline directories to be silently overwritten.",
     )
 
+    parser.add_argument(
+        "--allow-baseline-skip",
+        action="store_true",
+        default=default,
+        help="If the --generate option is given, then an attempt to overwrite "
```

Comment:
Add a note that this is incompatible with --allow-baseline-overwrite. And similarly add that this is incompatible with it.

Reply:
You already have that check in the code, just add a note about it here.

Reply:
Resolved with 9c8c2b31d.